qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -343,10 +343,13 @@ def _follow_selected_cb(self, js_elem, tab=False):
             tab: Open in a new tab.
         """
         if js_elem is None:
```

Comment:
Since we wan to fire `self.follow_selected_done` whenever we leave this (or the webkit) function, it might be a little cleaner to use contextmanagers. I don't know if it's worth the extra indentation level, but it would be a little less brittle to changes :P

Reply:
I ended up using a wrapper function with a try:/finally: instead. Good idea!